styles/globals.css

```
@@ -68,56 +164,860 @@
+
+/* HTTP verb -> pill, tinted from whatever colour fumadocs assigned.
+   Scoped tightly to the endpoint bar and the sidebar: the same
+   `font-mono font-medium` classes are used for parameter names, and
+   uppercasing those turns `bot_name` into `BOT_NAME`, which is wrong —
+   they are literal field names and must render exactly as written. */
+.api-reference .rounded-xl.border.bg-fd-card > span.font-mono.font-medium,
+#nd-sidebar span.font-mono.font-medium {
+  display: inline-flex;
+  align-items: center;
+  border-radius: 0.3rem;
+  border: 1px solid color-mix(in oklab, currentColor 32%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)